pkg/lightning/backend/local/local_unix.go

```diff
@@ -39,8 +39,8 @@ func GetSystemRLimit() (Rlim_t, error) {
 // In Local-backend, we need to read and write a lot of L0 SST files, so we need
 // to check system max open files limit.
 func VerifyRLimit(estimateMaxFiles Rlim_t) error {
-	if estimateMaxFiles < minRLimit {
-		estimateMaxFiles = minRLimit
+	if estimateMaxFiles > maxRLimit {
+		estimateMaxFiles = maxRLimit
```

Comment:
the "1024" introduced in pingcap/tidb-lightning#501 is no longer useful?

Reply:
Yes, `minRLimit` is to avoid the minlimit calculated too small, but it seems to be not very useful